scripts/runn.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
do we want to include this file in the repo?

Reply:
removed